### ISSUE TITLE
Further updates on install proceess and documentation

### ISSUE THREAD

**Issue by g7ruh** [dylski/PaperPiAI#4]
A few corrections to your recent updates:
(Humor):
> end of time (or a hardware failure - which ever comes first).

or 28 years but don't forget O/S bloat over time ... and the APT cache     :-)

Seriously:

Last para before Install: typo: interesting:

towards most intersting part of the square image

Enable E-paper interfaces should be "I2C"

In: Install Required Components:
should be "or" in " If you are building in a RPi 4 of 5 you can speed it up by" and "appending" has a typo

In: Storage:

"I simple fix is " do you mean A simple fix?

Update / observations on testing so far:

The compile works with swap file = 512. The most I saw was 410MB swap in use.

Completed the install process in 4 hour 8 minutes on PI Zero 2 W with a Raspberry Pi 32 GB sd card.

The last stage using git-lfs is very sensitive to the WiFi link signal strength. I moved the screen to a different position and got about a 2-fold increase in data transmission rates. That process took 1 hour to get to 90% complete then I moved the screen / pi 2 W. Given that the screen acts as a shield to WiFi placing the screen down exposed the WiFi antenna in a better direction towards the router box upstairs, directly above.

The final part of image generation causes the CPU to reduce clock speed because of high temperatures> over 80 Deg C. This is with a metal heatsink on the CPU.

Now going to try a Bookworm install.

Regards,
Roger G7RUH

**Comment by g7ruh** [comment]
Hi @dylski 

Further update on progress:

WiFi seems very sensitive to position of the screen and Pi Zero W 2. The screen has a lot of metal which attenuates the signal. Placing the Pi Zero W 2 facing the WiFi source makes a big difference to download speeds, given the amount of download needed to do the install. This impacts the total install time a lot. "A lot" will depend on signal strength but it is worth monitoring the download speed and moving the Pi / screen slightly to see if the download rate improves. Obviously, it only needs to be done once, but does add a lot to the total install time. The Pi 2 Zero W antenna is part of the circuit board and is not the most efficient (but it works), so  the position is a sensitive issue. This could make a large difference to the total install time.

I have installed on Bookworm Lite as per your instructions. No changes are needed for Bookworm / inky2. The default install gives the Bookworm version of the install files. It works the same as a Bullseye install.

I have done this on an SD card (the new Rasperry Pi cards which are faster than a lot of other cards) and on a 128 GB sandisk usb stick. Both work without problems as far as I can see. I have not detected any slow down, so far. The USB stick is a little faster as expected, but not significantly so.

There is still a significant heat issue during the latter part of image generation (decode), to the point where the CPU frequency is lowered due to high temperature (above 80 Deg C). The clock does go up again for a while as the temperature goes down (after the lower clock) and down again on temperature rise above 80 Deg C.

Swap Size needs to be set at minimum of 512 MB, may be 1024 to be safe, but I have had several installs complete at swap=512 (both Bullseye and Bookworm). I have only seen about 410 MB swap size in several tests. Of course a 1024MB swap reduces the available storage. On a 32 GB card it could drop below the 28 year storage mentioned previously   :)

Thank you for putting this package together. The images are very good, my wife likes them too.

I shall set this up to change images every so often (4 hours likely) over the Christmas festivities and see what reaction / feedback I get from family and visitors.

Hope the feedback helps.

Regards,

Roger G7RUH

**Comment by Nathanator** [comment]
Just wondering if it would work better as a container runtime on the Pi?  Happy to put in the work if needed.

**Comment by g7ruh** [comment]
Hi @Nathanator 

Just saw your suggestion: I am afraid I do not do containers yet. So I may be excluded if that happens.

I have yet to explore containers, but plan to do so. 

Would they add more CPU usage to the Pi Zero 2W? It gets pretty much overloaded CPU and memory-wise from what I have seen since I have been testing and enjoying @dylski creation. My wife likes the pictures too. Spouse-approval helps justify project time!

regards,

Roger G7RUH

**Comment by steph-sabotasan** [comment]
Might be helpful to specify min storage space recommended in the README?

**Comment by Nathanator** [comment]
> Just saw your suggestion: I am afraid I do not do containers yet. So I may be excluded if that happens.

That's the beauty of knowledge sharing! I wouldn't even consider it without quality documentation and a proven environment 😛

> Would they add more CPU usage to the Pi Zero 2W?

My quick back of the envelope mental gymnastics says that it shouldn't add more than a trivial amount. It would provide an easier, faster setup path. Also an easy way to do future upgrades to the stack.

**Comment by g7ruh** [comment]
> It would provide an easier, faster setup path. Also an easy way to do future upgrades to the stack.

Yes, that is why I want to dedicate time to learning how containers work and how much "bloat" they bring (if any), not withstanding the ease of deployment.

> That's the beauty of knowledge sharing!

Yes, thank you for your comments.

regards
Roger G7RUH
